packages/client/src/components/ProductCard/ProductCard.component.js

```diff
@@ -44,5 +44,8 @@ const ProductCard = ({ id, title, price, isFavorite, toggleFavorite }) => {
 ProductCard.propTypes = {
   title: PropTypes.string.isRequired,
   price: PropTypes.string.isRequired,
```

Comment:
This by the way, looks like a Product.

Reply:
I having issue with favorite page prop just merge in develop and now Its has changed props product name, I have mentioned in ProductPropType.js as name and favorite page mention props name as title.